app/ios/Flutter/ephemeral/flutter_lldbinit

```diff
@@ -0,0 +1,5 @@
+#
+# Generated file, do not edit.
+#
+
+command script import --relative-to-command-file flutter_lldb_helper.py
```

Comment:
Should this be included? The path says something about "ephemeral".

Reply:
I would just commit it. Running `flutter create temp` does not include `ios/Flutter/ephemeral` to `.gitignore`